src/js/components/form-elements/ProductSelectField.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import BaseField from 'components/form-elements/BaseField';
+import ProductSelect from 'components/product-select/ProductSelect';
+
+const ProductSelectField = (props) => {
+  // eslint-disable-next-line react/prop-types
```

Comment:
did you leave it on purpose?

Reply:
I don't think it's necessary anymore, I'll remove that